mu4/scenes/palette/palettemodule.cpp

```diff
@@ -20,19 +20,30 @@
 
 #include <QQmlEngine>
 
+#include "log.h"
+
 #include "config.h"
 #include "modularity/ioc.h"
 
 #include "internal/mu4paletteadapter.h"
+#include "internal/paletteconfiguration.h"
 
 #include "view/paletterootmodel.h"
 #include "internal/palette/paletteworkspace.h"
+#include "internal/palette/palettecreator.h"
+
+#include "workspace/iworkspacedatastreamregister.h"
+#include "workspace/iworkspacemanager.h"
+#include "internal/workspacepalettestream.h"
 
 #include "libmscore/score.h"
 #include "libmscore/sym.h"
 
 using namespace mu::scene::palette;
 
+static std::shared_ptr<MU4PaletteAdapter> m_adapter = std::make_shared<MU4PaletteAdapter>();
```

Comment:
Do we really need them to be shared?

Reply:
We register it in IoC, where it becomes shared, it is better that it was shared immediately